bundles/subclipse.core/src/org/tigris/subversion/subclipse/core/resourcesListeners/FileModificationManager.java

```diff
@@ -197,7 +197,7 @@ public void run() {
 							SVNProviderPlugin.broadcastModificationStateChanges(resources);
 						}
 					}
-				}, new RefreshStatusCacheSchedulingRule(MultiRule.combine(projectArray)), false);
```

Comment:
This change is basically undoing the `1a8bed9` change, which was made so that it would still be possible to save file changes in a project while a long running status cache update was going on. IMO, this fix needs to be done in a way that does not lose that functionality.

> When scheduling status cache refresh job use a scheduling rule that prevents
> other status cache refresh jobs from running at the same time for the same
> project (for example, if some tool is making changes in the workspace without
> wrapping the changes in a WorkspaceModifyOperation), but does not lock the
> project (i.e., does not prevent saving project changes).